File: snippets/fsharp/System/String/Substring/Substring4.fs
module Substring4.fs
// <Snippet4>
let value = "This is a string."
let startIndex = 5
let length = 2
let substring = value.Substring(startIndex, length)
printfn $"{substring}"

// The example displays the following output:
//       is
// </Snippet4>

File: snippets/csharp/System/String/Substring/Substring4.cs
using System;

public class SubstringLengthExample
{
    public static void Run()
    {
        // <Snippet4>
        string value = "This is a string.";
        int startIndex = 5;
        int length = 2;
        string substring = value.Substring(startIndex, length);
        Console.WriteLine(substring);

        // The example displays the following output:
        //       is
        // </Snippet4>
    }
}
